src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,9 +5,17 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
+    private int weight;
```

Comment:
you still don't have possibility to access these fields outside of the class, which is not quite good